### ISSUE TITLE
Slow performance, can't open more than 1 window, Android app unresponsive (new user)

### ISSUE THREAD

**Issue by philhzss** [dwainscheeren/dwains-lovelace-dashboard#603]
Hi,

I found out about this dashboard right when 2022.7 released so I didn't install it until yesterday (I was waiting for the 3.2.0 update). I finally got it all up and running but I'm having some serious performance issues. I don't think it's normal and something must be wrong on my end but I can't figure out what, because I'm a new Dwains Dashboard user and nothing is configured / customized yet.

Firstly, all this is running on an i7-10700 with 64GB of RAM. I noticed that since installing DD, I can no longer open more than 1 home assistant tab / page in Chrome, is this normal? I would usually have 2 or 3 Home Assistant windows open, so I can configure a script while looking at the dashboard for example. But now, if I have one window running DD, I cannot open any other Home Assistant tab, it just doesn't load until I close my first window.

This also seems to affect the Android app, I cannot open the Android app if I have a Home Assistant window open on my PC? 
If I open the Android companion app (which is still loading the lovelace regular dashboard by default), it'll say "unable to connect to Home Assistant", and if I press "wait", it takes about 60-90 seconds for the dashboard to load. Once it's loaded, everything seems fine... but if I close the companion app on my phone (or simply lock my screen), I have to wait for a minute once again next time I open Home Assistant. This also sometimes happens with 0 windows open on my PC, so perhaps it's unrelated to my "many windows open at once" problem.

Also, twice now I've had Chrome crash and say "out of memory" while browsing / editing DD, I look at task manager while just doing nothing inside a DD page and saw the RAM usage for Chrome was going up by ~50mb every second, until it reached 2GB and crashed. I have some kind of memory leak? This goes away completely if I uninstall DD.

So I had customized a few pages in DD but deleted everything, including the dwains_dashboard folder, uninstalled it from HACS, reinstalled, did 0 changes (I have the default non-configured DD right now) and I'm having the same issues.

Am-I doing something wrong, does anyone know what is going on?

Thanks a lot!

**Comment by davros123** [comment]
"all this is running on an i7-10700 with 64GB of RAM"
Is Home Assistant itself installed on that server ?

If so, no, none of what you are seeing is normal.

**Comment by philhzss** [comment]
Yes, Home Assistant is installed on that server. Well, it's a VM machine (running Proxmox with Ubuntu and HA is on a docker container on there), usually I only give HA 8GB of RAM and 4 CPU cores because it was plenty, but I gave it access to the entire 64GB of RAM + all the CPU cores from the i7-10700 and it didn't change anything, still having the same issue... I don't know what to change. 
Thanks for letting me know it's not normal

**Comment by dwainscheeren** [comment]
Thanks for reporting this.

Dwains Dashboard 3.9.0 has now been released, and the source files for DD3 are available in this repository.

DD3 is no longer under active feature development and will slowly be phased out. Active development continues in Dwains Dashboard Next:
https://github.com/dwainscheeren/dwains-dashboard-next

At the time of writing, Dwains Dashboard Next is still in alpha.

If this issue is a feature request, please check whether it fits Dwains Dashboard Next instead.

If this issue is a bug, please retest with DD3 version 3.9.0. If the bug is still reproducible there, please open a new issue with:

- Home Assistant version
- Dwains Dashboard version
- browser and device
- browser console errors
- clear reproduction steps

For more context, see #906.

For that reason, this older issue is being closed as part of the DD3 issue tracker cleanup.
